src/clients/FtpClient.cc

```diff
@@ -873,6 +864,8 @@ Ftp::Client::writeCommandCallback(const CommIoCbParams &io)
         /* failed closes ctrl.conn and frees ftpState */
         return;
     }
+
+    processControlReply();
```

Comment:
We cannot call it from Ftp::Client::writeCommand() even asynchronously because it can be called before the writer callback:
```
2025/09/26 15:54:56| FATAL: assertion failed: Write.cc:41: "!ccb->active()"
    current master transaction: master53
```